test/support/fixtures/asset.ex

```diff
@@ -1,8 +1,28 @@
 defmodule Teiserver.AssetFixtures do
+  require Logger
   alias Teiserver.Asset
   alias Teiserver.Repo
 
   def create_map(attrs) do
     %Asset.Map{} |> Asset.Map.changeset(attrs) |> Repo.insert!()
   end
+
+  def create_or_update_map(attrs) do
+    case Asset.get_map(attrs.spring_name) do
+      nil -> create_map(attrs)
+      map -> update_map(map, attrs)
+    end
```

Comment:
Thinking a bit more about that, I think that can work, but may create some interferences with other tests. For example, the ones like `teiserver/matchmaking/queue_test.exs`.
Another approach could be to have the name be randomly generated. So that with random queue_name, there's basically no chance of conflicts. Your call

Reply:
I was actually using randomised names but at the time it was a mess so I reverted the change and had a single map for a while.
Changed to use random map names again, simpler than updating existing ones and probably with less potential issues.